src/main/java/com/cvsgo/service/ReviewService.java

```diff
@@ -191,6 +197,42 @@ public Page<ReadProductReviewResponseDto> readProductReviewList(User user, Long
         return new PageImpl<>(results, pageable, totalCount);
     }
 
+    /**
+     * 리뷰 좋아요를 추가합니다.
+     *
+     * @param user     현재 로그인한 사용자
+     * @param reviewId 좋아요 하려는 리뷰 ID
+     */
```

Comment:
```suggestion
     * @throws NotFoundException  해당하는 아이디를 가진 리뷰가 없는 경우
     * @throws DuplicateException 이미 해당하는 리뷰 좋아요가 존재하는 경우
     */
```
리뷰 좋아요 삭제도 `@throws` 추가해주시면 될 것 같아요